src/test/kotlin/graphics/scenery/tests/examples/basic/SponzaExample.kt

```diff
@@ -44,7 +44,7 @@ class SponzaExample : SceneryBase("SponzaExample", windowWidth = 1280, windowHei
 
```

Comment:
Those changes do not belong here.

Reply:
I honestly have no idea how they got there

Reply:
addressed